stream-chat-android-ui-components/src/main/res/values/attrs_attachment_gallery_video_attachments.xml

```diff
@@ -49,6 +49,11 @@
         area of the gallery. -->
         <attr name="streamUiAttachmentGalleryViewMediaPlayVideoIconPaddingEnd"
             format="dimension" />
+        <!-- Sets the padding of the play video icon in the main viewing
+        area of the gallery. Using this attribute overrides other padding attributes such as
+        padding top, bottom, start and end.-->
+        <attr name="streamUiAttachmentGalleryViewMediaPlayVideoIconPadding"
```

Comment:
A convenience attribute introduced so that users who want equivalent padding on all sides don't have to override multiple settings.